CONTRIBUTING.md

```diff
@@ -51,16 +57,20 @@ Unfortunately, very little of Guppy is currently tested.
 We hope to add more tests in the meantime, as well as add CI integration to run tests on push, but for now you can run the tests with:
 
 ```
-npm run test
+yarn run test
 ```
 
 This project uses Flow, and the types can be checked with:
 
 ```
-npm run flow
+yarn run flow
 ```
 
-This project uses Prettier, this should be run automatically on commit.
+This project uses Prettier, this should be run automatically on commit. That step requires that you have yarn installed.
+
+### Debugging
+
+To help you debug, you can open the chromium developer tools inside the running Electron instance with `cmd+alt+i` (just like chrome). You can also open the redux developer tools with `ctrl+h` and close them with `ctrl+q`.
```

Comment:
> ⌥⌘I or <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd>
perhaps?
```markdown
⌥⌘I or <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd>
```

Reply:
wow, an entire new html element I had no idea about! And what an oddly specific one. Since the command is different between Mac/windows/linux I'll just sidestep that and refer to the View menu instead, which will list the platform specific shortcut.